README.md

```diff
@@ -46,35 +46,31 @@ And many more, please explore in [docs](https://pipecd.dev/docs).
 
 #
 
-### Quickstart
+### Getting Started
 
-The [quickstart guide](https://pipecd.dev/docs/quickstart/) shows how to set up PipeCD components and deploy a hello-world application with PipeCD.
+- The [quickstart guide](https://pipecd.dev/docs/quickstart/) shows how to set up PipeCD components and deploy a hello-world application with PipeCD for testing purposes.
+- The [installation guide](https://pipecd.dev/docs/installation/) explains and helps set up PipeCD for your real-life use environment.
 
 #
 
-### Installation
+### Community and development
 
-The [installation guide](https://pipecd.dev/docs/installation/) explains and helps set up PipeCD for your real-life production environment.
+- Check out the [PipeCD website](https://pipecd.dev) for the complete documentation and helpful links.
+- Join our [Slack workspace](https://cloud-native.slack.com/archives/C01B27F9T0X) to get help and to discuss features.
+- Tweet [@pipecd_dev](https://twitter.com/pipecd_dev) on Twitter/X.
+- Create Github [Issues](https://github.com/pipe-cd/pipecd/issues) or [Discussions](https://github.com/pipe-cd/pipecd/discussions/) to report bugs or request features.
+- Join our PipeCD Development and Community Meeting where we share the latest project news, demos, answer questions, and triage issues.
```

Comment:
How about move meetings' details to other site like Google Docs and just show its link here? 
We don't need to maintain README even if the details are changed.

Refer to flux: https://github.com/fluxcd/flux2?tab=readme-ov-file#community

Reply:
Make sense 👍 Addressed by https://github.com/pipe-cd/pipecd/pull/4848/commits/46ded8c5871a58bdc44aeac4b19134dd960b5cf3